examples/extensible-templates/views/A/a.njk

```diff
@@ -0,0 +1,6 @@
+{%- extends inheritance.testA -%}
```

Comment:
I don't think we need `-` here and in the other templates, what do you think about removing them to minify confusion?